src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -85,6 +85,41 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         $this->load($loader, $confDir, '/{services}_' . $this->environment);
     }
 
+    /**
+     * The "getContainerClass" need to be normalized for preview and over contexts
+     * as its used by the symfony cache component as prefix.
+     *
+     * @see https://github.com/symfony/symfony/blob/v4.4.7/src/Symfony/Component/Cache/DependencyInjection/CachePoolPass.php#L56
+     */
+    protected function getContainerClass()
+    {
+        return $this->generateContainerClass(static::class);
+    }
+
+    /**
+     * @internal
+     *
+     * This is only for internal use. To get the container class use `getContainerClass` instead.
+     *
+     * This is a copy of the symfony 5.0 getContainerClass which does not include $this->name.
```

Comment:
And where does this function include `$this->name`?

Reply:
> which does not include $this->name

The function from symfony 4.4 did include it this->name https://github.com/symfony/symfony/blob/0889df1e88559968eb291c938d25b96e252f7bed/src/Symfony/Component/HttpKernel/Kernel.php#L478

Not sure how to make this clear 🙈 

The function from symfony 5.0 does not longer include that name and thats why its copied so we have the behaviour from symfony 5 also for 4.4.

Reply:
Ok, I've misread :see_no_evil: